.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
     - CODENAME: reblochon
   matrix:
     - DOCKER_VERSION=1.9.1
-    - DOCKER_VERSION=1.10.1
+    - DOCKER_VERSION=1.12
```

Comment:
We should add a new docker version instead of replacing 1.10 one.
I think I handle the `tar.gz` thing in the docker file (but I might be wrong and could be in anothe project but…)